docs/enterprise-setup/scaling-airbyte.md

```diff
@@ -25,14 +29,34 @@ global:
   jobs:
     resources:
       limits:
-        cpu: 
-        memory: 
+        cpu: ## e.g. 250m
+        memory: ## e.g. 500m
       requests:
-        cpu: 
-        memory: 
+        cpu: ## e.g. 75m
+        memory: ## e.g. 150m
 ```
 
-If your Airbyte deployment is underprovisioned, you may often notice occasional 'stuck jobs' that remain in-progress for long periods, with eventual failures related to unavailable pods. If you begin to see such errors, we recommend you follow the troubleshooting steps above.
+If your Airbyte deployment is underprovisioned, you may notice occasional 'stuck jobs' that remain in-progress for long periods, with eventual failures related to unavailable pods. Increasing job CPU and memory limits may also allow for increased sync speeds.
+
+### Concurrent Sync Limits
+
+To help rightsize Airbyte deployments and reduce the likelihood of stuck syncs, there are configurable limits to the number of syncs that can be run at once:
+
+```yaml
+worker:
+  extraEnvs: ## We recommend setting a single value for both environment environment variables.
```

Comment:
"We recommend setting a single value for both environment environment variables." This might be less disorienting to read if reworded. Maybe something like "We recommend setting both environments with a single, shared value for the environment variable". This might not be necessary, because the example does help clarify.